CONTRIBUTING.md

```diff
@@ -105,11 +105,18 @@ curl -vvv -H "Authorization: Bearer $APPVEYOR_TOKEN" -XDELETE https://ci.appveyo
 
 ## Releasing
 
-First, kickstart a CI release to Sonatype by pushing a git tag.
+First, make sure that your current head is origin/master or reset to this branch
+```
+git reset --hard origin/master # you will loose the unpushed commits
+```
+
+Then, kickstart a CI release to Sonatype by pushing a git tag that correspond to the desired commit
 
 ```
+git log --pretty=oneline # choose the commit hash you want to tag
```

Comment:
You can list the remote commits without changing your local state

```suggestion
First, kickstart a CI release to Sonatype by pushing a git tag that correspond to the desired commit

git fetch && git log origin/master --pretty=oneline # choose the commit hash you want to tag
```

Reply:
is there any reason you wouldn't tag `origin/master` ?

Reply:
If some new commits just have been merged and they make more sense to be part later in an another release? 

It was just a proposition, maybe it doesn't make sense.